src/main/java/de/tum/in/www1/artemis/service/scheduled/cache/quiz/QuizScheduleService.java

```diff
@@ -293,17 +293,17 @@ public void startSchedule(long delayInMillis) {
     public void stopSchedule() {
```

Comment:
As far as I can tell the QuizScheduleService gets created on application start (see line 116.) Is this intentional or should it also be delayed?

Reply:
good point, I delayed it as well